src/prospectScraper/MockDraftPick.cs

```diff
@@ -33,36 +32,17 @@ public MockDraftPick(string pick, string team, string name, string school, strin
             this.school = school;
             this.position = pos;
             this.reachValue = relativeVal;
-            this.leagifyPoints = convertPickToPoints(pick, this.round);
+            this.leagifyPoints = ConvertPickToPoints(pick, this.round);
             this.pickDate = pickDate;
             this.state = GetState(school);
         }
+
         public static int ConvertPickToRound(string pick)
         {
-            // Compensatory picks added on 2/1/20
-            int intpick = 0;
-            var canParse = int.TryParse(pick, out intpick);
+            var canParse = int.TryParse(pick, out int intpick);
+
             if (canParse)
             {
-                /* 
```

Comment:
I'd prefer if this comment was kept, since this information is useful, and changes from year to year.

Reply:
Reverted